src/main/docs/guide/deployingTheApp.adoc

```diff
@@ -5,10 +5,4 @@ To deploy the app to Google App Engine run:
 $ ./gradlew appengineDeploy
 ----
 
-Initial deployment may take a while. When finished, you will be able to access your app:
-
-image::welcometograils.png[Welcome to Grails]
```

Comment:
any reasons you are removing the screenshots? I think that gives the user an idea if they are successful or not?